src/db/models/job.ts

```diff
@@ -48,7 +49,11 @@ export const JOB_METADATA_PROJECT_PDF = t.partial({
 
 export const JOB_METADATA_PROJECT_EXCEL = t.partial({
   startedBy: PARTICIPANT_ID,
-  fileName: t.string,
+  file: t.type({
+    fileHash: t.string,
+    name: t.string,
+    id: PLAN_ID,
```

Comment:
Using key `id` indicates this is file ID, whereas it is plan ID.
```suggestion
    planId: PLAN_ID,
```